### ISSUE TITLE
Side effect fix "ordered imports" + "quotemark" (single to double)

### ISSUE THREAD

**Issue by DanniB** [palantir/tslint#2940]
### Bug Report

- __TSLint version__: 5.4.3
- __TypeScript version__: 2.3.4
- __Running TSLint via__:  Angular-CLI 

#### TypeScript code being linted
Polyfills from [Angular Polyfills](https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/blueprints/ng/files/__path__/polyfills.ts)

```ts
import 'core-js/es6/symbol';
import 'core-js/es6/object';
import 'core-js/es6/function';
import 'core-js/es6/parse-int';
import 'core-js/es6/parse-float';
import 'core-js/es6/number';
import 'core-js/es6/math';
import 'core-js/es6/string';
import 'core-js/es6/date';
import 'core-js/es6/array';
import 'core-js/es6/regexp';
import 'core-js/es6/map';
import 'core-js/es6/weak-map';
import 'core-js/es6/set';
```

with `tslint.json` configuration:

 ```
{
    "rulesDirectory": [
        "node_modules/codelyzer"
    ],
    "rules": {
        "directive-selector": [true, "attribute", "sg", "camelCase"],
        "component-selector": [true, "element", "sg", "kebab-case"],
        "use-input-property-decorator": true,
        "use-output-property-decorator": true,
        "use-host-property-decorator": true,
        "no-attribute-parameter-decorator": true,
        "no-input-rename": true,
        "no-output-rename": true,
        "no-forward-ref": true,
        "use-life-cycle-interface": true,
        "use-pipe-transform-interface": true,
        "pipe-naming": [true, "camelCase", "sg"],
        "component-class-suffix": true,
        "directive-class-suffix": true,
        "templates-use-public": true,
        "invoke-injectable": true,
        "template-to-ng-template": true,

        "adjacent-overload-signatures": true,
        "align": [true, "arguments", "elements", "members", "parameters", "statements"],
        "class-name": true,
        "comment-format": [
            true,
            "check-space"
        ],
        "curly": true,
        "encoding": true,
        "forin": true,
        "indent": [
            true,
            "spaces"
        ],
        "label-position": true,
        "max-line-length": [
            true,
            150
        ],
        "max-file-line-count": [true, 1000],
        "member-access": false,
        "member-ordering": [
            true,
            {"order": "fields-first"}
        ],
        "no-angle-bracket-type-assertion": true,
        "no-arg": true,
        "no-bitwise": true,
        "no-conditional-assignment": true,
        "no-consecutive-blank-lines": true,
        "no-construct": true,
        "no-debugger": true,
        "no-duplicate-variable": true,
        "no-duplicate-super": true,
        "no-empty": false,
        "no-eval": true,
        "no-inferrable-types": true,
        "no-internal-module": true,
        "no-shadowed-variable": true,
        "no-string-literal": false,
        "no-switch-case-fall-through": true,
        "no-trailing-whitespace": true,
        "no-unused-expression": true,
        "no-use-before-declare": true,
        "no-var-keyword": true,
        "no-var-requires": true,
        "object-literal-key-quotes": [true, "as-needed"],
           wurden (Vorbeugen vor Merge-Konflikten)*/
        "object-literal-sort-keys": false,
        "one-line": [
            true,
            "check-open-brace",
            "check-catch",
            "check-else",
            "check-whitespace"
        ],
        "ordered-imports": true,
        "prefer-conditional-expression": true,
        "prefer-for-of": true,
        "prefer-switch": true,
        "prefer-template": true,
        "quotemark": [true, "double"],
        "radix": true,
        "semicolon": [true, "always"],
        "trailing-comma": [
            true, {"multiline": "never", "singleline": "never"}
        ],
        "triple-equals": [
            true,
            "allow-null-check"
        ],
        "typedef-whitespace": [true, {
            "call-signature": "nospace",
            "index-signature": "nospace",
            "parameter": "nospace",
            "property-declaration": "nospace",
            "variable-declaration": "nospace"
        }],
        "variable-name": [
            true,
            "ban-keywords"
        ],
        "whitespace": [
            true,
            "check-branch",
            "check-decl",
            "check-operator",
            "check-module",
            "check-separator",
            "check-type",
            "check-typecast",
            "check-preblock"
        ]
    }
}
```

#### Actual behavior

```
import "core-js/es6/symbol"
import '"core-js/es6/object"import 'co"core-js/es6/function"import 'co"core-js/es6/parse-int" 'core-js/"core-js/es6/parse-float"js/es6/num"core-js/es6/number"js/es6/obj"core-js/es6/math"e-js/es6/p"core-js/es6/string" 'core-js/"core-js/es6/date"mport 'cor"core-js/es6/array"import 'co"core-js/es6/regexp"ort 'core-"core-js/es6/map"import 'co"core-js/es6/weak-map"port 'core"core-js/es6/set";
```

#### Expected behavior

```
import "core-js/es6/array";
import "core-js/es6/date";
import "core-js/es6/function";
import "core-js/es6/map";
import "core-js/es6/math";
import "core-js/es6/number";
import "core-js/es6/object";
import "core-js/es6/parse-float";
import "core-js/es6/parse-int";
import "core-js/es6/regexp";
import "core-js/es6/set";
import "core-js/es6/string";
import "core-js/es6/symbol";
import "core-js/es6/weak-map";

**Comment by ajafff** [comment]
Looks like a duplicate of #2862 and should be fixed by #2864

**Comment by ajafff** [comment]
Closing this for housekeeping purposes. The fix landed months ago in v5.5.0. If there is still a problem, please add a comment.
